packages/atauth/src/auth_cli.c

```diff
@@ -503,7 +541,7 @@ exit: {
 
 // returns 1 if the error_message contains the ENROLLMENT_DENIED error code, otherwise 0
 int is_enrollment_denied(const char *err_msg) {
-  return strncmp(err_msg, ATAUTH_ENROLLMENT_DENIED_ERR_CODE, strlen(ATAUTH_ENROLLMENT_DENIED_ERR_CODE)) == 0 ? 1 : 0;
+  return strncmp(err_msg, ATAUTH_ENROLLMENT_DENIED_ERR_CODE, strlen(ATAUTH_ENROLLMENT_DENIED_ERR_CODE)) == 0;
 }
```

Comment:
For booleans, you can `#include <stdbool.h>` then return `true` or `false`, but this is fine.